params/config.go

```diff
@@ -487,6 +490,15 @@ func (c *ChainConfig) checkCompatible(newcfg *ChainConfig, head *big.Int) *Confi
 	return nil
 }
 
+// IsChaosCompatible checks whether consensus config of Chaos is compatible
+func (c *ChainConfig) IsChaosCompatible(newcfg *ChainConfig) bool {
+	if c.Chaos != nil && newcfg.Chaos != nil {
+		return c.Chaos.Period == newcfg.Chaos.Period && c.Chaos.Epoch == newcfg.Chaos.Epoch &&
+			c.Chaos.AttestationDelay == newcfg.Chaos.AttestationDelay
```

Comment:
What about the `c.Chaos.Rule`？

Reply:
`c.Chaos.Rule` is ignored for genesis compatibility which is already existed